test/unit/math/rev/mat/fun/cholesky_decompose_test.cpp

```diff
@@ -432,3 +437,21 @@ TEST(AgradRevMatrix, check_varis_on_stack_large) {
   X = stan::math::multiply(X, stan::math::transpose(X));
   test::check_varis_on_stack(stan::math::cholesky_decompose(X));
 }
+
+#ifdef STAN_OPENCL
+TEST(AgradRevMatrix, mat_cholesky_1st_deriv_large_gradients_opencl) {
+  stan::math::opencl_context.tuning_opts().cholesky_size_worth_transfer = 25;
+  test_gradient(51, 1e-08);
+  test_gp_grad(1300, 1e-08);
+  test_gp_grad(2000, 1e-08);
+  test_chol_mult(50, 1e-08);
+  test_simple_vec_mult(45, 1e-08);
+}
```

Comment:
`test_gradient`, `test_chol_mult`, and `test_simple_vec_mult` all evaluate the gradient `size` times. And I think they need to?

Is it cool to just lower the transfer to gpu size here so the GPU code goes off?